### ISSUE TITLE
Loop stuck in Initiated state

### ISSUE THREAD

**Issue by scarolan** [lightninglabs/lightning-terminal#318]
This is on a Raspiblitz device.

```
loop version 0.11.4-beta commit=v0.11.4-beta
```

I created a Loop via the UI and it's been stuck in initiated.  There are some failure messages in the log:

```
2022-02-02 21:43:33.982 [INF] LOOP: 8ba077 Waiting for either htlc on-chain confirmation or  off-chain payment failure
2022-02-02 22:10:05.460 [INF] LOOP: 8ba077 Server update: FAILED_OFF_CHAIN_TIMEOUT received, timestamp: 2022-02-02 22:10:03.366152201 +0000 GMT
2022-02-02 22:48:15.979 [INF] LOOP: 8ba077 Payment 15672274949f6015ba3d117a4f22de923be1d39e830430de18251aca1354431f: state=FAILED
2022-02-02 22:48:16.007 [INF] LOOP: 8ba077 Failed prepayment: payment failed
```

**Comment by scarolan** [comment]
Does anyone know how to cancel one of these stuck Loop operations?  I'm getting a little nervous as the on-chain and lightning balances are much smaller so there are satoshis that have gone missing or are stuck in limbo.

**Comment by guggero** [comment]
What `lnd` version are you using? Did you try restarting `lnd`? This sounds like https://github.com/lightningnetwork/lnd/issues/5396.

**Comment by scarolan** [comment]
Yes, I've restarted lnd.  The version is:

LND 0.12.1-beta commit=v0.12.1-beta

**Comment by guggero** [comment]
Try upgrading lnd to at least 0.13.3 and loop to the latest version as well. That might resolve the stuck payment.

**Comment by scarolan** [comment]
Can someone on the Lightning Labs side cancel this?  My raspiblitz has not been tested with 0.13.3 and I'm a bit nervous to upgrade to the so-called "reckless" version:

```
# reckless update option (latest LND release from GitHub)
lndLatestVersion='v0.14.2-beta'
lndLatestDownload='https://github.com/lightningnetwork/lnd/releases/download/v0.14.2-beta/lnd-linux-arm64-v0.14.2-beta.tar.gz'
```

**Comment by guggero** [comment]
`Server update: FAILED_OFF_CHAIN_TIMEOUT` means it was canceled on the server.

**Comment by scarolan** [comment]
Ok, so just the GUI is stuck. Did the transaction never begin then?  Is it safe to stop the `litd` service at this point?

**Comment by guggero** [comment]
Yes, the transaction was aborted and the payment canceled. It's just lnd that doesn't correctly update the state and the UI just shows the current state. It's safe to stop litd at this point, yes.

**Comment by scarolan** [comment]
Thank you so much, I will close this issue now.
